admin-dev/themes/new-theme/package-lock.json

```diff
@@ -12661,7 +12661,7 @@
                 "yargs-parser": {
                     "version": "11.1.1",
                     "resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-11.1.1.tgz",
-                    "integrity": "sha512-C6kB/WJDiaxONLJQnF8ccx9SEeoTTLek8RVbaOIsrAUS8VrBEXfmeSnCZxygc+XC2sNMBIwOOnfcxiynjHsVSQ==",
+                    "integrity": "sha1-h5oIZZc7yp9rq1y987HGfsfTvPQ=",
```

Comment:
What is the reason of these changes? Which node version are you using?

Reply:
```
$ node -v
v8.16.2
$ npm -v
6.4.1
```